101-hub-and-spoke-sandbox/azuredeploy.parameters.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
remove all non-GEN values from here - use the defaultValue in the template

Reply:
Done and acted on all other changes as well